py-polars/polars/series/series.py

```diff
@@ -4342,7 +4343,7 @@ def to_torch(self) -> torch.Tensor:
         # tensor.rename(self.name)
         return tensor
 
```

Comment:
I was thinking more of keeping `future: bool | None`, where we default to `None` and determine which version of pyarrow is available.

Reply:
An implicit logic depending on the version of an external dependency may render unexpected results and confuse users. And there may be also other arrow implementations besides libarrow. e.g., the arrow implementation (`polars-arrow`) that is statically linked to `pyo3-polars` plugins can be completely different from the pyarrow installed.

Reply:
The plugins are completely independent of what we decide to do in a `to_arrow` conversion. We are not paying the conversion to large utf8 if not needed. For the confusions, we could add it to the doctest. 

The confusion will also diminish as all future versions of pyarrow supprort utf8-view.